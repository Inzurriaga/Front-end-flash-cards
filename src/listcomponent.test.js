import React from 'react';
import ListComponent from './listcomponent.js';
import { shallow } from 'enzyme';

const cards = [{"catagory":"html","question":"What are the three main tags that all HTMl structure needs?","answer":"<html>, <head>, and <body>","falseAnswer":["<header>, <section>, and <footer>","<div>, <h1>, and <p>","<section>, <div>, and <span>"],"color":"#f1652a"},{"catagory":"html","question":"How do you insert an image?","answer":"<img>","falseAnswer":["<image>","<photo>","<jpeg>"],"color":"#f1652a"},{"catagory":"html","question":"What are the basic structure for a drop down menu?","answer":"<select> and <option>","falseAnswer":["<dropdown> and <option>","<input> and <option>","<input> and <dropdown>"],"color":"#f1652a"},{"catagory":"html","question":"How do you insert a video?","answer":"<video>","falseAnswer":["<vid>","<mp3>","<vhs>"],"color":"#f1652a"}];

describe('the list component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
        <ListComponent  cards={cards}/>);
  });

  it('should display the list', () => {
    expect(wrapper).toMatchSnapshot();
  });

});