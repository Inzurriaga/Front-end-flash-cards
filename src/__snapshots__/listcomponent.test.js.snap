// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the list component should display the list 1`] = `
<div
  className="list-component"
>
  <div>
    <h2>
      html
    </h2>
    <ul
      className="list"
    >
      <li
        className="list-card"
        style={
          Object {
            "background-color": "#f1652a",
          }
        }
      >
        <h4>
          html
        </h4>
        <p>
          What are the three main tags that all HTMl structure needs?
        </p>
        <p>
          &lt;html&gt;, &lt;head&gt;, and &lt;body&gt;
        </p>
      </li>
      <li
        className="list-card"
        style={
          Object {
            "background-color": "#f1652a",
          }
        }
      >
        <h4>
          html
        </h4>
        <p>
          How do you insert an image?
        </p>
        <p>
          &lt;img&gt;
        </p>
      </li>
      <li
        className="list-card"
        style={
          Object {
            "background-color": "#f1652a",
          }
        }
      >
        <h4>
          html
        </h4>
        <p>
          What are the basic structure for a drop down menu?
        </p>
        <p>
          &lt;select&gt; and &lt;option&gt;
        </p>
      </li>
      <li
        className="list-card"
        style={
          Object {
            "background-color": "#f1652a",
          }
        }
      >
        <h4>
          html
        </h4>
        <p>
          How do you insert a video?
        </p>
        <p>
          &lt;video&gt;
        </p>
      </li>
    </ul>
  </div>
  <div>
    <h2>
      css
    </h2>
    <ul
      className="list"
    />
  </div>
  <div>
    <h2>
      javascript
    </h2>
    <ul
      className="list"
    />
  </div>
  <div>
    <h2>
      personal
    </h2>
    <ul
      className="list"
    />
  </div>
</div>
`;
